go.mod

```diff
@@ -33,7 +33,7 @@ require (
 	github.com/gliderlabs/ssh v0.1.1
 	github.com/grafana/alloy/syntax v0.1.0
 	github.com/graph-gophers/graphql-go v1.6.0
-	github.com/mattermost/mattermost/server/public v0.1.11-0.20250317155315-c95968c380be
+	github.com/mattermost/mattermost/server/public v0.1.11-0.20250319140854-a87c4ec9be66
```

Comment:
This points to the PR branch for now: https://github.com/mattermost/mattermost/pull/30486

Reply:
What would happen when this new logic runs against a MM server without those changes?

Reply:
We probably need to version-guard some of the logic. Good point on that.